incubator/crate/templates/statefulset.yaml

```diff
@@ -0,0 +1,90 @@
+apiVersion: "apps/v1beta1"
+kind: StatefulSet
+metadata:
+  name: crate
+spec:
+  serviceName: "crate-db"
+  replicas: {{ .Values.crate.replicas }}
+  template:
+    metadata:
+      labels:
+        app: {{ .Values.app }}
+      annotations:
+        pod.alpha.kubernetes.io/initialized: "true"
+    spec:
+      initContainers:
+      - name: init-sysctl
+        image: busybox
+        imagePullPolicy: IfNotPresent
+        command: ["sysctl", "-w", "vm.max_map_count=262144"]
+        securityContext:
+          privileged: true
+      containers:
+      - name: crate
+        image: {{ .Values.image.name }}:{{ .Values.image.tag }}
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        command:
+          - /docker-entrypoint.sh
+          - -Clicense.enterprise=false
+          - -Ccluster.name=${CLUSTER_NAME}
```

Comment:
Good catch! no, this is a bug, copy-paste from our swarm recipes not properly updated. 

On a second thought, it's a bit redundant since we're already passing these values as env variables to the pod containers :/.